src/__tests__/SideMenu.test.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+import {SideMenu} from '../components/side-menu/SideMenu'
```

Comment:
For the import, we usually set before the import from node_modules, an empty line and the relative import. Can you check if there are other import like this in the PR, and change please?